packages/cozy-harvest-lib/src/models/konnector/KonnectorJobWatcher.js

```diff
@@ -66,6 +66,11 @@ export class KonnectorJobWatcher {
     this.emit('error', new KonnectorJobError(error))
   }
 
+  handleLoginSuccess() {
```

Comment:
don't we test that part?

Reply:
This would need to be an integration test. I think there is no point to add a unit test here.